source/tutorials/nix-shell-in-shebang.md

```diff
@@ -0,0 +1,33 @@
+# How to use nix-shel in a shebang
+
+The [nix
+shell](https://nixos.org/manual/nix/stable/command-ref/nix-shell.html)
+is feature provided by the [Nix package
+manager](https://nixos.org/guides/how-nix-works.html). It allows users
+to enter into a new shell environment containing a set of packages given
+on the command line.
+
+You can nix-shell as an interpreter](https://nixos.org/manual/nix/stable/command-ref/nix-shell.html#use-as-a--interpreter)
+in the shebang, not only to define the interpreter, but also a list the
+dependencies required for the script file.
```

Comment:
Depending on whether you want this to be a Tutorial or a Guide:
- for a Tutorial, the motivation for should come right after the page title, and followed by sections with a sequence of instructions (and detailed explanations where necessary)
- for a Guide, there should be only a very brief description of the feature, such as

```suggestion
Nix can [wrap interpreted scripts](https://nixos.org/manual/nix/stable/command-ref/nix-shell.html#use-as-a--interpreter) with a shebang to make them reproducible.
It allows you to precisely define all dependencies of a script, including the exact version of the interpreter.

Such a script will run with a working [Nix installation](install-nix) as its only prerequisite.
```

Reply:
I'll make it a tutorial, that's a nice entry to nix IMO